### PR TITLE
openvino_model_server release 2023.3 updates

### PR DESCRIPTION
**Pull request by z103cb**
Updates to the build configuration for openvino_model_server to switch releases branches

### CODE REVIEW

File: ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.3-release.yaml [hunk 1]
```diff
@@ -17,13 +17,13 @@ images:
   - name: BASE_OS
     value: redhat
   - name: PROJECT_VERSION
-    value: 2023.1.0
+    value: 2023.3.0
   - name: ov_source_branch
-    value: releases/2023/1
+    value: releases/2023/3
   - name: ov_source_org
     value: openvinotoolkit
```
Comment (Xaenalt) [inline ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.3-release.yaml:24]: We should swap the org here to opendatahub-io

**Review by z103cb — COMMENTED** [review]
> We should swap the org here to opendatahub-io 

Yes we will, as soon as when we create the branches in the respective repos.

**Pull request merged**